docs/thomson-mark.md

```diff
@@ -0,0 +1,4 @@
+# This is an introduction
+
```

Comment:
I really think this spaces is a great use of the negative to highlight contrast.